openedx/core/djangoapps/credit/tasks.py

```diff
@@ -0,0 +1,52 @@
+""" This file contains celery tasks for credit course views """
+
+from django.conf import settings
+
+from .api import set_credit_requirements
+from celery import task
```

Comment:
nitpick: separate third party imports from local import (add a new block for local imports).